app/src/main/java/com/github/se/orator/ui/profile/OfflineRecordingsProfile.kt

```diff
@@ -0,0 +1,130 @@
+package com.github.se.orator.ui.profile
+
+import android.content.Context
+import android.util.Log
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.ArrowBackIosNew
+import androidx.compose.material3.Card
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import com.github.se.orator.model.symblAi.SpeakingViewModel
+import com.github.se.orator.ui.navigation.NavigationActions
+import com.github.se.orator.ui.navigation.Screen
+import com.github.se.orator.ui.theme.AppDimensions
+import com.github.se.orator.ui.theme.AppTypography
+import loadPromptsFromFile
+
+@Composable
+fun PromptCard(
+    prompt: Map<String, String>,
+    index: Int,
+    navigationActions: NavigationActions,
+    speakingViewModel: SpeakingViewModel,
+    promptID: String
+) {
+  Card(
+      modifier =
+          Modifier.fillMaxWidth()
+              .height(AppDimensions.cardSectionHeight)
+              .padding(AppDimensions.paddingSmall)
+              .testTag("prompt_card_$index"),
+      onClick = {
+        speakingViewModel.interviewPromptNb.value = promptID
+        Log.d(
+            "OfflineRecordingsProfile: ",
+            "opening the file : ${speakingViewModel.interviewPromptNb.value}.mp3")
+        navigationActions.navigateTo(Screen.FEEDBACK_SCREEN)
+      }) {
+        Column(
+            modifier = Modifier.fillMaxSize().padding(AppDimensions.paddingMedium),
+            verticalArrangement = Arrangement.Center) {
+              Text(
+                  text = "Interview ${index + 1}",
+                  fontSize = 18.sp,
```

Comment:
You shouldn't be hardcoding the font size here

Reply:
fixed